WooCommerce/Classes/POS/Models/PointOfSaleAggregateModel.swift

```diff
@@ -413,9 +434,16 @@ extension PointOfSaleAggregateModel {
     func publishOrderState() {
         orderController.orderStatePublisher
             .map { $0.externalState }
-            .assign(to: &$orderState)
+            .sink(receiveValue: { [weak self] orderState in
```

Comment:
> Finally, replace as many .sink blocks in the aggregate model as I can – we should just be able to use simple properties, and can in most cases remove the reliance on publishers.

I'm intrigued by this point how we will remove the reliance on publishers. Will that require changes in the order controller as well or is there another way? 🤔

Reply:
Okay, https://github.com/woocommerce/woocommerce-ios/pull/15060 answers the question. 👍